test/mocks/lock.js

```diff
@@ -12,6 +12,7 @@ var waitingFibers = [];
 var MockLock = function() {
   this.hasLock_ = false;
   this.id = Math.random();
+  this.counter = 0;
```

Comment:
- The name `counter` could be a more descriptive and it's declaration could use a comment.
- A counter usually counts something, like `counter.count()`. Do you mean `count`?
- What about something more descriptive like `lockCount`?

Reply:
`MockProperties` and `MockCache` already use a variable with the same name. It's not part of an external API and I think the meaning is obvious enough.